modular_skyrat/master_files/code/modules/industrial_lift/tram_lift_master.dm

```diff
@@ -0,0 +1,2 @@
+/datum/lift_master/tram
+	collision_lethality = 4
```

Comment:
Could be nice to just add a comment here saying what range of damage that equates to, just so it's easier to understand why this was done this way in the future.

Reply:
Sounds good.